content-review/impl/turnitin-oc/pom.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<name>Content Review Turnitin OC Implementation</name>
+	<groupId>org.sakaiproject.contentreview.turnitin.oc</groupId>
+	<artifactId>content-review-turnitin-oc-impl</artifactId>
+	<packaging>jar</packaging>
+
+	<parent>
+		<groupId>org.sakaiproject.contentreview</groupId>
+		<artifactId>content-review-impl</artifactId>
+		<version>13-SNAPSHOT</version>
+	</parent>
+
+        <properties>
+                <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+                <jersey-version>1.18</jersey-version>
+                <maven-plugin-version>1.0.0</maven-plugin-version>
+        	<swagger-core-version>1.5.9</swagger-core-version>
+		<okhttp-version>2.7.5</okhttp-version>
+		<gson-version>2.6.2</gson-version>
+	</properties>
+
+	<dependencies>		
+		<dependency>
+			<groupId>org.sakaiproject.contentreview</groupId>
+			<artifactId>content-review-api</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.sakaiproject.contentreview.service</groupId>
+			<artifactId>content-review-service-impl</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.sakaiproject.kernel</groupId>
+			<artifactId>sakai-kernel-api</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.sakaiproject.kernel</groupId>
+			<artifactId>sakai-component-manager</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.sakaiproject.kernel</groupId>
+			<artifactId>sakai-kernel-util</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>net.sf.json-lib</groupId>
+			<artifactId>json-lib</artifactId>
+			<classifier>jdk15</classifier>
+		</dependency>
+		<dependency>
+			<groupId>org.sakaiproject.assignment</groupId>
+			<artifactId>sakai-assignment-api</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.httpcomponents</groupId>
+			<artifactId>httpclient</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.httpcomponents</groupId>
+			<artifactId>httpmime</artifactId>
+		</dependency>
+        <dependency>
```

Comment:
Can you clean up tabs/spaces in this file?

Reply:
updated